pcdsdevices/areadetector/detectors.py

```diff
@@ -63,6 +64,28 @@ def get_plugin_graph_edges(self, *, use_names=True, include_cam=False):
                           for src, dest in port_edges]
         return port_edges
 
+    def screen(self, main=False):
+        """
+        Open camViewer screen for camera.
```

Comment:
This is actually the second camviewer open script in this file, I think your implementation may be better than the previous.
https://github.com/pcdshub/pcdsdevices/blob/22c5b9cbfcb3330d0a698d1230903fea26e1dd24/pcdsdevices/areadetector/detectors.py#L361-L372